### ISSUE TITLE
Problem in flexPaperViewer.SwfFile = url

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fleszar1/flexpaper#91]
```
In the new version (1.3) I had to comment line 269 of Viewer.as to read my swf.
Read my swf dynamically via a URL.

/ * If (_libMC == null) (if (_libMC.parent! = Null) (_libMC.parent.removeChild 
(_libMC);) _libMC.unload ();) * / _libMC = null;

```

Original issue reported on code.google.com by `wpso...@gmail.com` on 21 Jun 2010 at 1:07

**Comment by GoogleCodeExporter** [comment]
```
if(_libMC!=null){if(_libMC.parent!=null){_libMC.parent.removeChild(_libMC);}_lib
MC.unload();}_libMC = null;
```

Original comment by `yiditu...@163.com` on 21 Jun 2010 at 4:04

**Comment by GoogleCodeExporter** [comment]
```
Thus only the first swf and read.
```

Original comment by `wpso...@gmail.com` on 21 Jun 2010 at 5:55

**Comment by GoogleCodeExporter** [comment]
```
should be fixed in code branch 1.3.1
```

Original comment by `erik.eng...@gmail.com` on 21 Jun 2010 at 7:49
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
will build 1.3.1 and release
```

Original comment by `erik.eng...@gmail.com` on 21 Jun 2010 at 7:50
